src/less/toast/base/toast.less

```diff
@@ -0,0 +1,97 @@
+:root {
+    --ebay-skin-toast-spacing: 24px;
+}
+
+.toast__mobile {
+    --ebay-skin-toast-spacing: 16px;
+}
+
+.transition() {
+    transition:
+        opacity 0.2s cubic-bezier(0.21, 0.31, 1, 1.22) 0s,
+        transform 0.2s cubic-bezier(0.21, 0.31, 1, 1.22) 0s;
+}
+
+.toast {
+    background-color: @toast-box-color;
+    bottom: 16px;
+    color: @white-color;
+    left: 16px;
+    max-height: 40vh;
+    max-width: 480px;
+    min-width: 320px;
+    position: fixed;
+    z-index: 1;
+
+    &.toast--transition {
+        display: block;
+        transform: translateY(110%);
+    }
+}
+
+.toast.toast__mobile {
+    bottom: 0;
+    left: 0;
+    width: 100vw;
+}
+
+.toast.toast--transition.toast--hide {
+    opacity: 0;
+    transform: translateY(110%);
+
+    .transition();
+}
+
+.toast.toast--transition.toast--show {
+    opacity: 1;
+    transform: translateY(0);
+
+    .transition();
+}
+
+.toast__header {
+    align-items: center;
```

Comment:
Use the `.dialog-header-content-large()` or without large. Feel free to add selectors as you need like for the `h3` to that mixin.